### DISCUSSION TITLE
How can i access entity pages?

### DISCUSSION THREAD

**Discussion by JohannesRegner** [Blackymas/NSPanel_HA_Blueprint#1074] [Q&A]
Hi,
i configured an etitiy page with the Blueprint v4, but there is no icon on the home screen, how can i access the entity page, for power consumption in example... Thanks

**Comment by edwardtfn** [comment]
On the Blueprint setting there's a toggle to enable entity pages. Is that enabled?

**Reply by JohannesRegner** [reply]
Yes, it is... in the config yaml is this:

    entitypages_enabled: true
    entity_page01_label: Energieverbrauch
    entities_entity01: sensor.zaehlerhausodr
    entities_entity01_name: Test

**Reply by edwardtfn** [reply]
So you should have an icon in the home page for opening the entities pages.
I'm not at home now, so u can't send a picture, but it looks like a list with bullet points.

**Reply by edwardtfn** [reply]
![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/1353a3e4-0c7d-48eb-bb33-6b9ac1b7add1)

**Reply by pjoaquin66** [reply]
Hello, I have the same issue. Any Icon appear on right side: qr code, entities, ... 
I have marked the true parameter in entitypages_enabled of blueprint automation.

**Reply by edwardtfn** [reply]
On boot screen, which versions are shown for TFT, ESPHome and Blueprint?

**Reply by JohannesRegner** [reply]
Hmm the icon isn't showing. ESPHome and Blueprint version 4.0 ...

**Reply by edwardtfn** [reply]
What about TFT version?
